src/main/java/IncomeTaxCalculator.java

```diff
@@ -53,10 +56,12 @@ public void calculateIncomeTax(double taxPaid) {
     public static void main(String[] args) {
         System.out.println("Have you paid too much income tax? Let's find out! \n" +
                         "Do you have your P60?");
+        //resources
         String p60 = new Scanner(System.in).next();
         if (p60.equalsIgnoreCase("yes") || p60.equalsIgnoreCase("Y")) {
             System.out.println("For the following questions, please use the information provided on your P60. \n" +
                      "What was your total annual salary last year?");
+            //resuse?
```

Comment:
There's no need to create a new Scanner every time, you can reuse the existing one.  Because they establish connections to system resources, in this case the console, they are quite expensive items to create, so you don't really want to make more than you have to.